randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -140,12 +142,19 @@ Extra - asset_id: 3
   * Normal Door to West Ocean/Door to Moat
   > Door to Crateria Kihunter Room
       Any of the following:
-          # Via top path - TODO: check for shinespark without entrance rando?
-          Grapple Beam or Space Jump
+          Any of the following:
+              # Via top path
+              Grapple Beam or Space Jump or Movement (Intermediate)
+              # Start in West Ocean and Jump Over with momentum
+              Speed Booster
```

Comment:
If you start in west ocean, then  no entrance rando + no door lock rando needs to be disabled

Reply:
Done.